rholang/src/main/scala/interpreter/Proc.scala

```diff
@@ -159,6 +163,10 @@ object Proc {
     }
   }
 
+  /* 
```

Comment:
Use this style for multi-line comments:
```
 /**
  * Differs from [...]
  * cancels to n.
  */
```

See:  https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards